apps/android/app/src/main/java/chat/simplex/app/model/ChatModel.kt

```diff
@@ -550,7 +552,7 @@ data class ChatItem (
       id: Long = 1,
       dir: CIDirection = CIDirection.DirectSnd(),
       ts: Instant = Clock.System.now(),
-      text: String = "hello\nthere",
+      text: String = generalGetString(R.string.hello_there__multiline),
```

Comment:
I don't think we need to localise examples/tests, it would unnecessarily increase the number of strings to maintain.